src/App.jsx

```diff
@@ -1,45 +1,15 @@
 import React from 'react';
 import './App.scss';
+import { Person } from './components/Person/Person';
 
-export const misha = {
-  name: 'Misha',
-  age: 37,
-  sex: 'm',
-  isMarried: true,
-  partnerName: 'Natasha',
-};
-
-export const olya = {
-  name: 'Olya',
-  sex: 'f',
-  isMarried: true,
-  partnerName: 'Maksym',
-};
-
-export const alex = {
-  name: 'Alex',
-  age: 25,
-  sex: 'm',
-  isMarried: false,
-};
+import { misha } from './components/People/Misha';
+import { olya } from './components/People/Olya';
+import { alex } from './components/People/Alex';
```

Comment:
I think it's not the best solution. Looks better if persons stay in this file. You can be merged them in one object. But this fix is not required, optional to realize.